src/tests/unit/tests/common/components/copy-issue-details-button.test.tsx

```diff
@@ -66,16 +76,14 @@ describe('CopyIssueDetailsButtonTest', () => {
                 })
                 .verifiable(Times.once());
 
-            const result = Enzyme.mount(<CopyIssueDetailsButton {...props} />);
-            const button = result.find(DefaultButton);
+            const result = render(<CopyIssueDetailsButton {...props} />);
             onClickMock.setup(m => m(It.isAny())).verifiable(Times.once());
             // tslint:disable-next-line: await-promise
-            await button.simulate('click');
-
-            const toast = result.find(Toast);
+            await userEvent.click(result.container.querySelector('.ms-Button-label'));
```

Comment:
For this one, it's worth it to not mock `DefaultButton` and use `result.getByRole('button')` for clarity.

Reply:
resolved the comment